devops/dependencies.json

```diff
@@ -19,9 +19,9 @@
       "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
     },
     "level_zero": {
-      "github_tag": "v1.8.12",
-      "version": "v1.8.12",
-      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.8.12",
+      "github_tag": "v1.11.0",
+      "version": "v1.11.0",
+      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.11.0",
```

Comment:
I will suggest to update the loader in a separate PR. We have seen issues in the past where just updating the loader introduces issues, so having it in a separate PR would be easier for future debug.